server/index.js

```diff
@@ -1,4 +1,17 @@
 const express = require('express')
 const app = express()
+const helpers = require('../database/index.js');
```

Comment:
This variable name could probably better

Reply:
Hi Toby, I am on it.